server.go

```diff
@@ -806,6 +806,7 @@ func (s *Server) incrCallsFailed() {
 }
 
 func (s *Server) sendResponse(t transport.ServerTransport, stream *transport.Stream, msg interface{}, cp Compressor, opts *transport.Options, comp encoding.Compressor) error {
+	beginTime := time.Now()
```

Comment:
Note that `t.Write` doesn't actually write the request on wire. `transport` has a queue of messages, and `Write` only pushed the message to the queue. So returning from `Write` doesn't mean the message is sent.

The gap between `begin` and `HandleRPC` actually only measures the time spent on encoding and compression.